pkgs/tools/misc/nmfd/default.nix

```
@@ -0,0 +1,26 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  name = "nmfd-${version}";
```

You can do

```suggestion
  pname = "nmfd";
```

nowadays